src/cpip-pack

```diff
@@ -74,7 +74,7 @@ install_compilers()
 
         # install compilers
         local pkgs=("$cc_pkg" "$cxx_pkg" "$gfortran_pkg" "$swig_pkg" "$ld_impl" "$pkg_config")
-        local pkg_diff="$(conda install --override-channels -c anaconda ${pkgs[@]} --no-update-deps --json --quiet)"
+        local pkg_diff="$(conda install --override-channels -c anaconda -c conda-forge ${pkgs[@]} --no-update-deps --json --quiet)"
```

Comment:
To reduce the ammount of changes in this PR what do you think about 1) removing conda-forge and 2) setting swig to 4.0.2
I mention this because I want to avoid adding the potential probs that adding another channel may bring